vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/TouchSteps.java

```diff
@@ -98,7 +99,7 @@ public void tapByLocator(Locator locator)
      * Swipes to element in <b>direction</b> direction with duration <b>duration</b>
      * The step takes into account current context. If you need to perform swipe on the element,
      * you need to switch the context to this element.
-     * @param direction direction to swipe, either <b>UP</b> or <b>DOWN</b>
+     * @param direction direction to swipe, either <b>UP</b> or <b>DOWN</b> or <b>LEFT</b> or <b>RIGHT</b>
```

Comment:
please, update the docs as well

Reply:
docs have already been updated:

<img width="871" alt="docs" src="https://user-images.githubusercontent.com/18533593/198702138-c84ff317-1950-4d92-8456-a6daf7f3c14d.png">